patches/async-transaction.patch

```diff
@@ -0,0 +1,257 @@
+Adds methods to perform transactions asynchronously.
+
+From: Saxon Parker <saxon@bestateless.com>
+
+
+---
+ GNUmakefile            |    4 +++
+ src/Transaction.cc     |   48 ++++++++++++++++++++++++++++++--
+ src/Transaction.h      |   21 ++++++++++++--
+ src/TransactionTest.cc |   73 ++++++++++++++++++++++++++++++++++++++++++++++++
+ 4 files changed, 142 insertions(+), 4 deletions(-)
+
+diff --git a/GNUmakefile b/GNUmakefile
+index 50d491f6..201131cc 100644
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -431,7 +431,10 @@ INSTALL_INCLUDES := \
+     src/BoostIntrusive.h \
+     src/Buffer.h \
+     src/ClientException.h \
++    src/ClientTransactionManager.h \
+     src/CodeLocation.h \
++    src/Common.h \
++    src/Context.h \
+     src/CoordinatorClient.h \
+     src/CoordinatorRpcWrapper.h \
+     src/Crc32C.h \
+@@ -467,6 +470,7 @@ INSTALL_INCLUDES := \
+     src/SpinLock.h \
+     src/Status.h \
+     src/TestLog.h \
++    src/Transaction.h \
+     src/Transport.h \
+     src/Tub.h \
+     src/WireFormat.h \
+diff --git a/src/Transaction.cc b/src/Transaction.cc
+index c12867e3..8fb1f342 100644
+--- a/src/Transaction.cc
++++ b/src/Transaction.cc
```

Comment:
Several trailing whitespace issues on added lines in this fie.

Reply:
donesies